packages/pluggable-widgets-tools/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+
+- We updated our library versions (React and React Native), please be aware this might break your widgets. New libraries versions: React: 18.0.2, React-Native: 0.70.7
+
 ## [9.23.2] - 2023-03-07
```

Comment:
It needs a changelog for the generator-widgets too

Reply:
Added